### PR TITLE
feat(frontend): add arrow icon on campaign hover

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

**Review by guerrap — APPROVED** [review]

File: packages/frontend/src/components/campaigns/index.tsx [hunk 1]
```diff
@@ -86,6 +141,16 @@ export function Campaigns() {
                     value={search}
                     onChange={handleSearchChange}
                 />
+                <Select
+                    options={statusOptions}
+                    value={status}
+                    onChange={handleStatusChange}
+                    placeholder="Status"
```
Comment (guerrap) [inline packages/frontend/src/components/campaigns/index.tsx:148]: We should use the i18n keys here.

File: packages/frontend/src/components/campaigns/index.tsx [hunk 2]
```diff
@@ -86,6 +141,16 @@ export function Campaigns() {
...
+                    messages={{
+                        noResults: "no",
```
Comment (guerrap) [inline packages/frontend/src/components/campaigns/index.tsx:150]: Same here.

File: packages/frontend/src/ui/select/styles.module.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    @apply flex
+        justify-center
+        items-center
+        p-3
+        h-12;
+}
+
+.option {
+    @apply cursor-pointer
+        p-3
```
Comment (guerrap) [inline packages/frontend/src/ui/select/styles.module.css:26]: Add the usual transition classes (duration, ease, ...).

File: packages/frontend/src/components/campaigns/index.tsx (hunk 2 above)
Comment (luzzif) [reply packages/frontend/src/components/campaigns/index.tsx]: I just left the empty string there. The options are hardcoded so we will never see this.

File: packages/frontend/src/ui/select/styles.module.css (hunk 3 above)
Comment (luzzif) [reply packages/frontend/src/ui/select/styles.module.css:26]: Addressed in aa2e218191c5a74d76060d9f84949c7e9ac1e843

**Pull request merged**